sematic/api/server.py

```diff
@@ -183,27 +189,27 @@ def parse_arguments() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def run_wsgi(daemon: bool):
-    options = {
-        "bind": f"{get_config().server_address}:{get_config().port}",
-        "workers": get_config().wsgi_workers_count,
-        "worker_class": "geventwebsocket.gunicorn.workers.GeventWebSocketWorker",
-        "daemon": daemon,
-        "pidfile": get_config().server_pid_file_path,
-        "logconfig_dict": make_log_config(log_to_disk=True),
-        "certfile": os.environ.get("CERTIFICATE"),
-        "keyfile": os.environ.get("PRIVATE_KEY"),
-    }
+def app():
+    dictConfig(make_log_config(log_to_disk=True, level=logging.DEBUG))
```

Comment:
Same log level comment.

Reply:
same answer; don't mean to merge like this (was for dev). Changing to INFO